catalog/app/components/JsonEditor/Column.js

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react'
+import isArray from 'lodash/isArray'
```

Comment:
I used these a lot. What you think if I will refactor it later and use `R.is` insead of lodash's `isArray`, `isObject` and so on?

Reply:
Please, elaborate on why you don't like lodash in this case, because there are a several reasons that I can imagine. And they affect code differently.

Would you accept this:
Module `utils/types.js` with a number of functions `isNull`, `isArray` and so on. Internally, every function will be has no external dependency (like `isNull(x) => x === null`) or maybe `Ramda` dependency

Reply:
> Please, elaborate on why you don't like lodash in this case

well, the main reason is we already use ramda and it has most of this functionality, so i see no point in importing extra code that does the same.

as for the utils module, idk but i think it's not necessary, bc `isNull` is such a trivial thing that i dont think we should abstract it away, just simple `(x) => x === null` or `R.equals(null)` seems good enough. other cases like `isObject` and `isString`, for example, are trivial as well with ramda, it's just `R.is(Object)`, `R.is(String)` and so on -- this, again, seems like it's good enough.

`isArray` is a special case, because we have a built-in function for this -- `Array.isArray`, so we don't need to use any abstractions or introduce any dependencies for this.

however, i think using `lodash` is legit when we want some special logic that's unique to it (missing in ramda)

Reply:
I created a backlog task for that in the Code Quality section